crm_rma_stock_location/models/product.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+# © 2014-2016 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+import openerp.addons.decimal_precision as dp
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    rma_qty_available = fields.Float(
+        compute='_rma_product_template_available',
+        digits_compute=dp.get_precision('Product Unit of Measure'),
+        string='RMA Quantity On Hand'
+    )
+    rma_virtual_available = fields.Float(
+        compute='_rma_product_template_available',
+        digits_compute=dp.get_precision('Product Unit of Measure'),
+        string='RMA Forecasted Quantity'
+    )
+
+    def _rma_product_template_available(self):
+        """ Compute rma_qty_available and rma_virtual_available
+        with sum of variants quantities.
+        """
+
+        for template in self:
+            qantities = template.product_variant_ids.read(
+                ['rma_qty_available', 'rma_virtual_available']
+            )
+            template.rma_qty_available = sum(
+                qty['rma_qty_available'] for qty in qantities
+            )
+            template.rma_virtual_available = sum(
+                qty['rma_virtual_available'] for qty in qantities
+            )
+
+
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
+
+    rma_qty_available = fields.Float(
+        compute='_rma_product_available',
+        digits_compute=dp.get_precision('Product Unit of Measure'),
+        string='RMA Quantity On Hand'
+    )
+    rma_virtual_available = fields.Float(
+        compute='_rma_product_available',
+        digits_compute=dp.get_precision('Product Unit of Measure'),
+        string='RMA Forecasted Quantity'
+    )
+
+    def _rma_product_available(self):
```

Comment:
None, anyway it depends partly of values in the context. I would leave it empty.